.gitignore

```diff
@@ -313,3 +313,6 @@ third_party/gflags/
 
 # Test output file logs to ignore
 stats_log
+
+test.txt
```

Comment:
This should not be included into the pull request to the master, right... cuz it is for own specific testing use.